### PR TITLE
Changed the color of light-color-text and highlighted text in …

### LINKED ISSUE #9192: Change dark theme colour of highlighted text in Entry Merge Dialogue
JabRef 5.8--2022-08-22--a255311
Linux 5.4.0-126-generic amd64 
Java 18.0.2 
JavaFX 18.0.2+2

**Problem:**

- text is hard to read
- distinction between red-green is not good for people with [impaired colour vision](https://en.wikipedia.org/wiki/Color_blindness)

grafik
grafik

**Describe the solution you'd like**

Please use complementary colour schemes for the fix. See here: https://en.wikipedia.org/wiki/Complementary_colors

### PR DESCRIPTION
**Pull request by Asrk03**
…entry merge dialogue

Fixes #9192

**Before:**

grafik
grafik

**After:**
Screenshot 2023-01-14 235730
Screenshot 2023-01-15 004947
Screenshot 2023-01-15 005014
Screenshot 2023-01-15 005032

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [x] ~Tests created for changes (if applicable)~
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - In the context of a systematic literature review (SLR), we reworked the "Define study" parameters dialog. [#9123](https://github.com/JabRef/jabref/pull/9123)
 - We upgraded to Lucene 9.4 for the fulltext search. The search index will be rebuild. [#9213](https://github.com/JabRef/jabref/pull/9213)
 - We disabled the "change case" menu for empty fields. [#9214](https://github.com/JabRef/jabref/issues/9214)
-- We disabled the conversion menu for empty fields. [#9200](https://github.com/JabRef/jabref/issues/9200)
+- We disabled the conversion menu for empty fields. [#9200](https://github.com/JabRef/jabref/issues/9200)
```
Comment (k3KAW8Pnf7mkmdSMPHz27) [inline CHANGELOG.md:95]: Why is there an extra white space here?

```suggestion
- We disabled the conversion menu for empty fields. [#9200](https://github.com/JabRef/jabref/issues/9200)
```

Comment (Asrk03) [reply CHANGELOG.md]: Got it @k3KAW8Pnf7mkmdSMPHz27, removed the white space.

**Comment by Siedlerchr** [comment]
Thanks for your work, I am not that happy with the chosen colors. 
I think maybe something similar to the search color? Maybe you can also check some contras colors https://coolors.co/contrast-checker/

Search highlight color: 
grafik

**Comment by Asrk03** [comment]
@Siedlerchr thank you for your response, will make the changes soon.

**Comment by Siedlerchr** [comment]
Thanks for the updated colors! Looks better now for me. Let's ask the others. @JabRef/developers

**Comment by ThiloteE** [comment]
image

I think light yellow and light orange do not give the best contrast, but oh well. Definitely better than before! I personally find it easier to read now. Thanks!

image

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
- I like the change of text color to black

If we are interested in a discussion, I liked the dull red/sky blue for delete/add, but perhaps a higher contrast blue, maybe #006dff :stuck_out_tongue: 

Anyway, I agree that this is a clear improvement

**Review by k3KAW8Pnf7mkmdSMPHz27 — APPROVED** [review]
Just for clarity, I am in favor of merging the PR and perhaps discuss further during the dev-call?

**Comment by ThiloteE** [comment]
How about these three colours?

   - #ADD, #AADDDD, or rgb(170,221,221)
   - #FEC, #FFEECC, or rgb(255,238,204)
   - #F99, #FF9999, or rgb(255,153,153)
    
Source: https://en.wikipedia.org/wiki/File:Safe_Chart_Colors-F99-FEC-ADD.jpg

Was taken from the wikipedia entry about colour blindness.
https://en.wikipedia.org/wiki/Color_blindness

**Comment by Asrk03** [comment]
Screenshot 2023-01-14 211223
Screenshot 2023-01-14 211301
Screenshot 2023-01-14 210904
Screenshot 2023-01-14 212338

Thank you for your suggestions!
Please let me know which one you like. I will make respective changes.
Personally I like 1st better here.

**Comment by Siedlerchr** [comment]
I would be happy with 2 or 3

**Comment by calixtus** [comment]
How about the red from 4 with the orange from 3?

**Comment by Asrk03** [comment]
@calixtus I think this combo is not very pleasing to eyes
Screenshot 2023-01-14 235730

**Comment by Asrk03** [comment]
I think in this image the texts are discernable and the colors contrast well with each other too.
Should I finalize these colors?
image

**Comment by Siedlerchr** [comment]
I agree, I would go for the last and original version: https://github.com/JabRef/jabref/pull/9546#issuecomment-1382887447

**Comment by Asrk03** [comment]
I have made the changes please review them @Siedlerchr @k3KAW8Pnf7mkmdSMPHz27 @ThiloteE

**Review by Siedlerchr — APPROVED** [review]

**Comment by Siedlerchr** [comment]
Thanks again for your contributions and the updates!

**Comment by claell** [comment]
Sorry, I am late. Right now, it seems that the meaning of the colors are no longer there. At least in git, it is common to use red for deletions and green for additions (and neutral if nothing changed). I feel it is sensible to preserve that scheme, no? I can also open a new issue for that in case this is preferred.

**Comment by claell** [comment]
From Visual Studio Code (https://code.visualstudio.com/docs/sourcecontrol/overview):

grafik

**Comment by ThiloteE** [comment]
The red green combo is particular problematic for people with colour blindness.

**Comment by claell** [comment]
Yes, I saw that afterward. The thing is that the colors are used here to show information. So it might be hard to keep track of the changes when they are not red or green, I think. But I haven't tested, probably one can get used to that. I need to test that before making further statements on that.

When I upload the screenshot from VS Code to a color blindness simulator, the shades can still be differentiated; although not as much as the ones right now.

**Comment by claell** [comment]
@ThiloteE the user reference above isn't working. Has the user left GitHub or is there some spelling mistake?

**Comment by ThiloteE** [comment]
Yes, she has deleted her account.

**Comment by koppor** [comment]
> I can also open a new issue for that in case this is preferred.

Yes, please do. Please try to collect the information provided here in a structured form. Especially [MADR](https://adr.github.io/madr/) could be of help here. Feel free to open a pull request with the different options (and links of them). Then we can decide and then move forward to an implementation.

Reasoning: For a quick reader, it is very hard to follow the discussion here and to be able to decide something. Is it just about changing a color back to something or is it about a general UI design of JabRef? - Refs https://github.com/JabRef/jabref/issues/7771.
